packages/nimble-components/src/chip/specs/README.md

```diff
@@ -0,0 +1,200 @@
+# Nimble Chip
+
+## Overview
+
+The `nimble-chip` is a component that presents text, an optional "remove" button, and optional slotted content to the left of the text.
+
+### Background
+
+- [Nimble Chip component issue](https://github.com/ni/nimble/issues/413)
+- [Chip design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=9269-229459&m=dev)
+- [Spright chat feature backlog](https://github.com/ni/nimble/issues/2610) (component for attachment chips)
+
+### Containing Library
+
+The intent is for this to be put in Nimble, for the following reasons:
+
+- This is a fairly constrained component, and thus easily ported to either Angular or Blazor.
+- It has broad use possibilities.
+- Designs for the component have been completed (see above link)
+- While the remove button introduces a new interactive pattern for our controls from an accessibility perspective, the described plan seems in-line with ARIA expectations.
+
+### Non-goals
+
+- Arbitrary HTML content
+- Implementing a "chip container" component that would be responsible for laying out the chips in a particular fashion, as well as possibly managing their removal if the "remove" button on a chip is pressed. While this may be useful at some point it is not necessary for the `Chip` component to have value.
+- Usage guidance will be provided in the Storybook.
```

Comment:
> Usage guidance will be provided in the Storybook.

Is this a comment associated with the bullet point directly above? If so, perhaps the sentence should be added to the above bullet point or as a sub-bullet.

As-is, it's unclear how saying you'll add usage documentation in storybook is considered a non-goal.

Reply:
Yeah, I've just removed this line.